web/beacon-app/cypress/e2e/TopicDetail/TopicDetail.ts

```diff
@@ -0,0 +1,185 @@
+import { And, Given, Then, When } from 'cypress-cucumber-preprocessor/steps';
+
+Given("I'm logged into Beacon", () => {
+    cy.fixture('user').then((testUser) => {
+        cy.loginWith( {email: testUser.email, password: testUser.password });
+    });
+});
+
+And("I navigate to the Topic Detail Page", () => {
+    cy.contains('div', 'Projects').should('exist').click();
+    cy.location('pathname').should('include', 'app/projects')
+    cy.get('[data-cy="projectTable"]').within(() => {
+        cy.get('tr>td').eq(0).click()
+    })
+    cy.location('pathname').should('include', 'app/projects/01H0924MJW1QB8JKT30PP815ET')
```

Comment:
It would be highly beneficial if we could obtain the ProjectID parameters from fixture data. This approach would enable me to conduct testing on my end without needing to directly modify the code.